reactive/jenkins.py

```diff
@@ -179,7 +178,7 @@ def update_plugins():
     update_interval = time.time() - (config("plugins-auto-update-interval") * 60)
     if (last_update < update_interval):
         status_set("maintenance", "Updating plugins")
-        remove_state("jenkins.updated.plugins")
+        remove_state("jenkins.configured.plugins")
```

Comment:
This remove_state and the set_state down a bit on 194 can be removed I think, as it is a noop. After unsetting the flag, there is no early return and the set_state is always called, and no change has happened. If a failure occurs and an exception raised, the remove_state will never get committed (unless the exception is caught, which it isn't).

Reply:
I was trying to use that to control when the state gets back to active just like it's done in configure_plugins, but I removed those and it works the same way. Now I understand why.
Addressed in 02e4884.